recipe/meta.yaml

```
@@ -14,10 +14,11 @@ source:
 build:
   number: 0
 
-  skip: true  # [py==27, py<39]
+  skip: true  # [py==27, py<310]
```

This syntax is not correct. I think you mean just:

```suggestion
  skip: true  # [py<310]
```

---

Ok. I also need some help with the re-render process. WHen I rerender the recipe the process deletes a whole bunch of necessary files and breaks things. See https://conda.discourse.group/t/conda-forge-re-rendering-of-a-feedstock-removed-necessary-files-in-ci-support/626

---

It was rerendering to no files because the syntax was wrong. A rerender now is working correctly.